Makefile

```diff
@@ -4,6 +4,63 @@ install_dir ?= $(root_dir)/utils/bin
 
 SBT ?= sbt
 SBT_FLAGS ?= -Dsbt.log.noformat=true
+MKDIR ?= mkdir -p
+CURL ?= curl -L
+MILL_BIN ?= $(HOME)/bin/mill
+# The following must correspond with the equivalent definition in build.sc
+ANTLR4_JAR ?= $(HOME)/lib/antlr-4.7.1-complete.jar
```

Comment:
This doesn't appear to be true. `ANTLR4_JAR` in `build.sc` is
```scala
val ANTLR4_JAR = (pwd / s"antlr-${antlr4Version}-complete.jar").toString
```
which puts `ANTLR4_JAR` somewhere down in firrtl.
The attempt to download (for me) fails with:
```
[15/117] firrtl[2.12.4].generatedSources 
Downloading ANTLR4 .jar
1 targets failed
firrtl[2.12.4].generatedSources ammonite.ops.ShelloutException: CommandResult 1
error(5):  directory not found: /Volumes/ucbjrlNBP/jrl/clients/ucb/git/ucb-bar/firrtl/firrtl/src/main/antlr4
error(7):  cannot find or open file: /Volumes/ucbjrlNBP/jrl/clients/ucb/git/ucb-bar/firrtl/firrtl/src/main/antlr4/FIRRTL.g4

    ammonite.ops.Shellout$.executeStream(Shellout.scala:101)
```
Notice the `.../firrtl/firrtl/...` path segment.